src/controllers/article.js

```diff
@@ -10,6 +10,8 @@ exports.get = (req, res, next) => {
         style: ['article', 'header', 'footer', 'public'],
         javascript: ['hambruger'],
         article: result.rows[0],
+        login: (res.locals.unlockCookie === null),
```

Comment:
so if there is no cookie then this would evaluate to true!
so he would be marked as login without having a cookie
shouldn't be !==null